tests/Model/Client.php

```diff
@@ -6,8 +6,6 @@
 
 class Client extends User
 {
-    public $table = 'client';
```

Comment:
well, I am ok with table here - it is probably how you should model Models, it was a question if this change is related with refactoring (and therefore a separate PR for it is better/clearer) or needed with this PR

Reply:
No breaking changes but anyway I moved setup to the new code